internal/pkg/docker/dockerfile/ignorefile.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package dockerfile
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/moby/patternmatcher/ignorefile"
+	"github.com/spf13/afero"
+)
+
+// DockerignoreFile represents a .dockerignore file holding information of excluded files.
+type DockerignoreFile struct {
```

Comment:
Why do we make a new struct instead of just having a public function that takes `fs`, and `contextDir`, and returns `[]string` a list of dockerignore files to ignore?

Reply:
I think I thought I needed it to do the mocks, but yes this can be removed and cleans up the code a ton. Thanks!